aspnet/fundamentals/hosting.rst

```diff
@@ -0,0 +1,297 @@
+:version: 1.0.0
+
+Hosting
+=======
+
+By `Steve Smith`_
+
+To run an ASP.NET Core app, you need to configure and launch a host using `WebHostBuilder <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html?highlight=webhostbuilder>`_.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+What is a Host?
+---------------
+
```

Comment:
This is an odd staring point as nobody ever works with an IWebHost, just the WebHostBuilder